src/Symfony/Component/Debug/Tests/Fixtures/ToStringThrower.php

```diff
@@ -16,9 +16,11 @@ public function __toString()
         try {
             throw $this->exception;
         } catch (\Exception $e) {
-            // Using user_error() here is on purpose so we do not forget
-            // that this alias also should work alongside with trigger_error().
-            return user_error($e, E_USER_ERROR);
+        } catch (\Throwable $e) {
```

Comment:
Mayb I should add another fixture `ToStringThrowableThrower` and add a test for it?